docs/get-started/query-analytics.md

```diff
@@ -218,6 +218,29 @@ The *Plan* tab shows the plan for PostgreSQL queries (only available when using
 
 ![!image](../_images/PMM_Query_Analytics_Tabs_Plan.png)
 
+## Query Analytics for PostgreSQL
+
+In QAN we supporting two types of query source. Its `pg_stat_monitor` and `pg_stat_statements`. In the past PGSS were default query source, but since this version its newly PGSM. 
```

Comment:
In QAN we do support two types of query sources, `pg_stat_monitor` and `pg_stat_statements`. In the past PGSS was the default query source, but since this version, it's now PGSM.